gobblin-metastore/src/test/java/org/apache/gobblin/metastore/testing/TestMetastoreDatabaseServer.java

```diff
@@ -58,20 +54,22 @@ class TestMetastoreDatabaseServer implements Closeable {
 
   public static final String CONFIG_PREFIX = "gobblin.metastore.testing";
   public static final String EMBEDDED_MYSQL_ENABLED_KEY = "embeddedMysqlEnabled";
-  public static final String EMBEDDED_MYSQL_ENABLED_FULL_KEY =
-      CONFIG_PREFIX + "." + EMBEDDED_MYSQL_ENABLED_KEY;
+  public static final String EMBEDDED_MYSQL_ENABLED_FULL_KEY = CONFIG_PREFIX + "." + EMBEDDED_MYSQL_ENABLED_KEY;
   public static final String DBUSER_NAME_KEY = "testUser";
-  public static final String DBUSER_NAME_FULL_KEY =  CONFIG_PREFIX + "." + DBUSER_NAME_KEY;
+  public static final String DBUSER_NAME_FULL_KEY = CONFIG_PREFIX + "." + DBUSER_NAME_KEY;
   public static final String DBUSER_PASSWORD_KEY = "testPassword";
-  public static final String DBUSER_PASSWORD_FULL_KEY =  CONFIG_PREFIX + "." + DBUSER_PASSWORD_KEY;
+  public static final String DBUSER_PASSWORD_FULL_KEY = CONFIG_PREFIX + "." + DBUSER_PASSWORD_KEY;
   public static final String DBHOST_KEY = "dbHost";
-  public static final String DBHOST_FULL_KEY =  CONFIG_PREFIX + "." + DBHOST_KEY;
+  public static final String DBHOST_FULL_KEY = CONFIG_PREFIX + "." + DBHOST_KEY;
   public static final String DBPORT_KEY = "dbPort";
-  public static final String DBPORT_FULL_KEY =  CONFIG_PREFIX + "." + DBPORT_KEY;
+  public static final String DBPORT_FULL_KEY = CONFIG_PREFIX + "." + DBPORT_KEY;
+  public static final String MYSQL_VERSION = "8.0.20";
+  public static final String MYSQL_ROOT_PASSWORD = "MYSQL_ROOT_PASSWORD";
+  private final String charSet = "utf8mb4";
+  private final String collate = "utf8mb4_unicode_ci";
```

Comment:
Since these are constants, they can be private static final

From https://gobblin.apache.org/docs/developer-guide/CodingStyle/

> Constant variables are declared as static final and should be all uppercase ASCII letters delimited by underscore ("_"), e.g. FOO_BAR

Reply:
addressed